Makefile

```diff
@@ -82,6 +86,9 @@ pd-server-failpoint:
 	FAILPOINT=1 $(MAKE) pd-server || { $(FAILPOINT_DISABLE); exit 1; }
 	@$(FAILPOINT_DISABLE)
 
+pd-server-plugin:
+	PLUGIN=1 $(MAKE) pd-server
+
```

Comment:
Do you mean remove this command instead of original `PLUGIN=1 $(MAKE) pd-server`? @lhy1024

Reply:
yes